README.md

```diff
@@ -1,63 +1,58 @@
-    <p align="center">
-<a href="https://www.npmjs.com/~nestjscore" target="_blank"><img src="https://img.shields.io/npm/v/@nestjs/core.svg" alt="NPM Version" /></a>
-<a href="https://www.npmjs.com/~nestjscore" target="_blank"><img src="https://img.shields.io/npm/l/@nestjs/core.svg" alt="Package License" /></a>
-<a href="https://www.npmjs.com/~nestjscore" target="_blank"><img src="https://img.shields.io/npm/dm/@nestjs/common.svg" alt="NPM Downloads" /></a>
-<a href="https://circleci.com/gh/nestjs/nest" target="_blank"><img src="https://img.shields.io/circleci/build/github/nestjs/nest/master" alt="CircleCI" /></a>
-<a href="https://coveralls.io/github/nestjs/nest?branch=master" target="_blank"><img src="https://coveralls.io/repos/github/nestjs/nest/badge.svg?branch=master#9" alt="Coverage" /></a>
-<a href="https://discord.gg/G7Qnnhy" target="_blank"><img src="https://img.shields.io/badge/discord-online-brightgreen.svg" alt="Discord"/></a>
-<a href="https://opencollective.com/nest#backer" target="_blank"><img src="https://opencollective.com/nest/backers/badge.svg" alt="Backers on Open Collective" /></a>
-<a href="https://opencollective.com/nest#sponsor" target="_blank"><img src="https://opencollective.com/nest/sponsors/badge.svg" alt="Sponsors on Open Collective" /></a>
-  <a href="https://paypal.me/kamilmysliwiec" target="_blank"><img src="https://img.shields.io/badge/Donate-PayPal-ff3f59.svg"/></a>
-    <a href="https://opencollective.com/nest#sponsor"  target="_blank"><img src="https://img.shields.io/badge/Support%20us-Open%20Collective-41B883.svg" alt="Support us"></a>
-  <a href="https://twitter.com/nestframework" target="_blank"><img src="https://img.shields.io/twitter/follow/nestframework.svg?style=social&label=Follow"></a>
-</p>
-  <!--[![Backers on Open Collective](https://opencollective.com/nest/backers/badge.svg)](https://opencollective.com/nest#backer)
-  [![Sponsors on Open Collective](https://opencollective.com/nest/sponsors/badge.svg)](https://opencollective.com/nest#sponsor)-->
-
 ## Description
 
-[Nest](https://github.com/nestjs/nest) framework TypeScript starter repository.
+NestJS JWT API boilerplate.
+
+# Installation
+
+## Prerequisites
+
+- [`Node`](https://nodejs.org/en/download) v12 LTS
+- Globally [`yarn`](https://yarnpkg.com/cli/install)
+- [`Docker`](https://docs.docker.com/get-docker)
 
-## Installation
+Install yarn packages before continue
 
 ```bash
-$ npm install
+yarn
 ```
 
-## Running the app
-
 Ask other developers to share `.development.env`. For security reasons this file is not versioned.
 
+## Setting up PostgreSQL database
+
+- This is will make a new PostgreSQL running in the standard port `5432`
+- Please shutdown any previous conflicting PostgreSQL instances before starting
+  this
+
 ```bash
-# development
-$ npm run start
+docker-compose up
+```
+
+Check the database is up
+
+```bash
+docker logs -f postgres
+```
+
+Go to http://localhost:5433 to acces pgadmin UI.
```

Comment:
There is a typo here:

```suggestion
Go to http://localhost:5433 to access pgadmin UI.
```

Reply:
Resolved on https://github.com/DeyvidHolz/nestjs-jwt/pull/5/commits/e3f54acd320aac1f2cfe8d89dfa8f01affdbd74c